examples/Python/MandelbrotSet/MandelbrotSet.pyde

```diff
@@ -0,0 +1,71 @@
+"""
+MandelbrotSet
+https://rosettacode.org/wiki/Mandelbrot_set#Processing
+Processing 3.4
+2017-09-23 Noel
+2020-03-13 Alexandre (Python Mode)
+Task: Generate and draw the Mandelbrot set.
+Note that there are many algorithms to draw Mandelbrot set
+and there are many functions which generate it .
+"""
+
+# Click on an area to zoom in.
```

Comment:
Thanks for doing this! Basically the style is that the wiki task instructions all go in the header comments, and any notes by the sketch author about the specific solution go in line comments.